src/truststore/_windows.py

```diff
@@ -240,14 +240,17 @@ def _handle_win_error(result: bool, _: Any, args: Any) -> Any:
 
 
 def _verify_peercerts_impl(
-    cert_chain: list[bytes], server_hostname: str | None = None
+    ssl_context: ssl.SSLContext,
+    cert_chain: list[bytes],
+    server_hostname: str | None = None,
 ) -> None:
     pCertContext = None
     ppChainContext = None
     hStore = CertOpenStore(CERT_STORE_PROV_MEMORY, 0, None, 0, None)
     try:
-        # Add intermediate certs to store
-        for cert_bytes in cert_chain[1:]:
+        # Add intermediate and loaded certs to store
+        ctx_certs: list[bytes] = ssl_context.get_ca_certs(binary_form=True)  # type: ignore[assignment]
+        for cert_bytes in cert_chain[1:] + ctx_certs:
```

Comment:
Again, I think we need to not just get them into the same place as the certs from the server, but also instruct Windows to consider them trusted. oscrypto (MIT-licensed) goes through some contortions to do that here: https://github.com/wbond/oscrypto/blob/master/oscrypto/_win/tls.py#L597

I haven't figured out yet if there's a different way to do it by telling Windows to consider the cert to be a trusted root cert at the time of adding it to the store or getting the cert chain instead of telling it to ignore the fact that the root cert isn't trusted while verifying the chain.